go.mod

```diff
@@ -0,0 +1,21 @@
+module github.com/Financial-TimesFinancial-Times/methode-content-placeholder-mapper/v2
```

Comment:
Typo: `github.com/Financial-TimesFinancial-Times/methode-content-placeholder-mapper/v2` should be `github.com/Financial-Times/methode-content-placeholder-mapper/v2`